### ISSUE TITLE
linux kernel 6.8 / 6.9 compile problems

### ISSUE THREAD

**Issue by geosone** [EnzephaloN/ncpfs-module#15]
`>>> Emerging (1 of 2) net-fs/ncpfs-modules-9999::linamh
 * Fetching files in the background.
 * To view fetch progress, run in another terminal:
 * tail -f /var/log/emerge-fetch.log
 * Determining the location of the kernel source code
 * Found kernel source directory:
 *     /usr/src/linux
 * Found sources for kernel version:
 *     6.9.8-xanmod1-dist
 * Checking for suitable kernel configuration options ...                                                                                                                                                  [ ok ]
>>> Unpacking source...
 * Repository id: EnzephaloN_ncpfs-module.git
 * To override fetched repository properties, use:
 *   EGIT_OVERRIDE_REPO_ENZEPHALON_NCPFS_MODULE
 *   EGIT_OVERRIDE_BRANCH_ENZEPHALON_NCPFS_MODULE
 *   EGIT_OVERRIDE_COMMIT_ENZEPHALON_NCPFS_MODULE
 *   EGIT_OVERRIDE_COMMIT_DATE_ENZEPHALON_NCPFS_MODULE
 * 
 * Fetching https://github.com/EnzephaloN/ncpfs-module.git ...
git fetch https://github.com/EnzephaloN/ncpfs-module.git +refs/heads/master:refs/heads/master
git symbolic-ref refs/git-r3/net-fs/ncpfs-modules/0/__main__ refs/heads/master
 * Checking out https://github.com/EnzephaloN/ncpfs-module.git to /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
git checkout --quiet master
GIT update -->
   repository:               https://github.com/EnzephaloN/ncpfs-module.git
   at the commit:            bf610b91c5f7c4adf4af0814fc08cba61ee4bef9
>>> Source unpacked in /var/tmp/portage/net-fs/ncpfs-modules-9999/work
>>> Preparing source in /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
ln: failed to create symbolic link 'Module.symvers': File exists
 * Preparing ncpfs module
make -j12 -s HOSTCC=x86_64-pc-linux-gnu-gcc 'LDFLAGS=-m elf_x86_64' KDIR=/usr/src/linux M=/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 all 
/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999/dir.c:87:6: warning: no previous prototype for ‘new_dentry_update_name_case’ [-Wmissing-prototypes]
   87 | void new_dentry_update_name_case(struct dentry *dentry, const struct qstr *name)
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999/inode.c: In function ‘init_inodecache’:
/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999/inode.c:88:49: error: ‘SLAB_MEM_SPREAD’ undeclared (first use in this function)
   88 |                                                 SLAB_MEM_SPREAD|SLAB_ACCOUNT),
      |                                                 ^~~~~~~~~~~~~~~
/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999/inode.c:88:49: note: each undeclared identifier is reported only once for each function it appears in
make[3]: *** [scripts/Makefile.build:244: /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999/inode.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [/usr/src/linux-6.9.8-xanmod1-dist/Makefile:2070: /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999] Error 2
make[1]: *** [Makefile:240: __sub-make] Error 2
make: *** [Makefile:49: all] Error 2
`

**Comment by hundertzwei** [comment]
Kannst Du bitte mal den Patch in meinen Pullrequest ausprobieren?

**Comment by geosone** [comment]
jep funzt.

**Comment by geosone** [comment]
` * Checking out https://github.com/hundertzwei/ncpfs-module.git to /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
git checkout --quiet kernel-6.9
GIT NEW branch -->
   repository:               https://github.com/hundertzwei/ncpfs-module.git
   at the commit:            f695298ba6200cfc689249d42522e0772a3751af
>>> Source unpacked in /var/tmp/portage/net-fs/ncpfs-modules-9999/work
>>> Preparing source in /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 ...
ln: failed to create symbolic link 'Module.symvers': File exists
 * Preparing ncpfs module
make -j12 -s HOSTCC=x86_64-pc-linux-gnu-gcc 'LDFLAGS=-m elf_x86_64' KDIR=/usr/src/linux M=/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999 all 
/var/tmp/portage/net-fs/ncpfs-modules-9999/work/ncpfs-modules-9999/dir.c:87:6: warning: no previous prototype for ‘new_dentry_update_name_case’ [-Wmissing-prototypes]
   87 | void new_dentry_update_name_case(struct dentry *dentry, const struct qstr *name)
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~
>>> Source compiled.
>>> Test phase [not enabled]: net-fs/ncpfs-modules-9999
`

**Comment by EnzephaloN** [comment]
Merged into master
